lexicons/app/bsky/graph/getSuggestedFollowsByActor.json

```diff
@@ -0,0 +1,33 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.graph.getSuggestedFollowsByActor",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Get suggested follows related to a given actor.",
+      "parameters": {
+        "type": "params",
+        "required": ["actor"],
+        "properties": {
+          "actor": {"type": "string", "format": "at-identifier"},
+          "limit": {"type": "integer", "minimum": 1, "maximum": 100, "default": 50},
```

Comment:
do we need pagination on this? it becomes even easier if we remove the need for an ordered result set & just send back results with no promise of order or completeness